halconfig/settings.js

```diff
@@ -98,6 +99,7 @@ window.spinnakerSettings = {
   defaultTimeZone: timezone, // see http://momentjs.com/timezone/docs/#/data-utilities/
   defaultCategory: 'serverGroup',
   defaultInstancePort: 80,
+  maxPipelineAgeDays: 14,
```

Comment:
ah, i think this would need to be `maxPipelineAgeDays: maxPipielineAgeDays` so that it makes use of the injected variable on line 25.  Is that right, @lwander?

Reply:
ah whoops, thanks! need more coffee!